EmployeeList/Models/Employee.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Employee.swift
+//  EmployeeList
+//
+//  Created by Wang Siman on 1/29/21.
+//  Copyright © 2021 None. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+enum EmployeeType: String, Decodable {
+    case FULL_TIME
```

Comment:
I would make these into swift-style enums (fullTime, partTime, etc) and then give them strings for the actual values.

Reply:
The Square team describes the employee_type field in JSON like this:

employee_type : enum (FULL_TIME, PART_TIME, CONTRACTOR) 

A JSON sample:
```
{
  "employees" : [
    {
      "uuid" : "some-uuid",
      "full_name" : "Eric Rogers",
      "phone_number" : "5556669870",
      "email_address" : "erogers.demo@squareup.com",
      "biography" : "A short biography describing the employee.",

      "photo_url_small” : "https://some.url/path1.jpg",
      "photo_url_large” : "https://some.url/path2.jpg",

      "team" : “Seller",
      "employee_type" : "FULL_TIME",
    },

    ...

  ]
}
```

I don't know how to give those enums the raw values though. And I found that declaring EmployeeType cases of String type and using uppercase names are easy to make EmployeeType codable. Could you show my the way to make them swift-style?

Reply:
```enum EmployeeType: String, Codable { case fullTime = "FULL_TIME", partTime = "PART_TIME", contractor = "CONTRACTOR" }```